docs/features/authorized-keys.rst

```diff
@@ -13,3 +13,13 @@ If you select this package, add a list of authorized keys to ``site.conf`` like
     ...
 
 Existing keys in ``/etc/dropbear/authorized_keys`` will be preserved.
+
+If you want to remove specific keys in the future, specify them like this:::
```

Comment:
Take note here that this will also remove SSH key which are originally added by the user.